docs/source/modeling_with_aequilibrae/project_database/transit_graph.rst

```diff
@@ -0,0 +1,9 @@
+Transit graph configuration
+===========================
+
+The 'transit_graph_configs' table holds information on the configuration parameters for a 
+``TransitGraph`` of a particular 'period_id'.
+
+.. seealso::
+
+    * :func:`aequilibtae.transit.TransitGraphBuilder`
```

Comment:
This link is not working

Reply:
I am actually not sure if this fit here at all, as this is not really a "Project database" component

Reply:
Yeah, you're right that it doesn't fit in "project database". The reason why I added it here is simply because it is created in the project database and it is a table created and documented under the project database umbrella. 

Maybe we can remove this file and add it later when rebuilding the documentation. What do you think about it?